src/Bridge/LeagueOpenAPIValidation/Factory.php

```diff
@@ -62,4 +62,59 @@ public function getResponseValidator(): ResponseValidator
             throw new InvalidOpenApiDefinitionException($error);
         }
     }
+    
+    /**
+     * OpenApi doc is internally transposed by Raven to a tree of infos.
+     * This method gives the opportinity to retreive a subtree, based on the "path" in the api doc
+     *  that is represented by the tupple: http method / uri / status code / content type
+     * One usage can be to further process the resulting array
+     *  in order to check an api json response object's structure conformity
+     */
+    public function getReferenceStructure(string $method, string $uri, int $statusCode, ?string $contentType): ?array
```

Comment:
Isn't that method recreating the `League\OpenAPIValidation\PSR7\PathFinder` logic ? This object is able to retrieve (based on method and uri) a list of available operations.